.gitignore

```diff
@@ -2,7 +2,7 @@
 build
 coverage
 cypress
-hooks
+/hooks
```

Comment:
is this intended?

Reply:
Yes, we want to exclude only top level folder. Not, all folders named hooks. We might want to revisit also other rules in this file. I run on this issue when working on coediting when git skipped subfolder named hooks.